storage/innobase/dict/dict0stats.cc

```diff
@@ -44,6 +45,32 @@ Created Jan 06, 2010 Vasil Dimov
 #include <vector>
 #include <thread>
 
+/** Gets the externally stored size of a record, in units of a database page.
+@param[in]	rec	record
+@param[in]	offsets	array returned by rec_get_offsets()
+@return externally stored part, in units of a database page */
+static uint32_t rec_get_n_blob_pages(
+  const rec_t *rec, const rec_offs *offsets) noexcept
```

Comment:
Please remove the redundant `[in]`, and do not split the line before the first parameter of the function. Lines ending in `(` were only accepted in the old InnoDB formatting style.